manifests/install.pp

```diff
@@ -75,13 +75,20 @@
     $_package_options = {}
   }
 
+  if $::osfamily =~ /windows/ {
```

Comment:
OK I started plumbing out passing the `local_package_file_path` in from `init` and it is going to touch a lot more than this change.